### PR TITLE
Messenger Kernel List test

### PR DESCRIPTION
**Pull request by Kelvinrr**
This tests the function that returns available Mess kernels + some fixes caught by the tests.

### CODE REVIEW

File: SugarSpice/tests/QueryTests.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  ASSERT_EQ(res["mdis"]["ck"]["deps"]["sclk"].size(), 2);
+  ASSERT_EQ(res["mdis"]["ck"]["deps"]["lsk"].size(), 2);
+  ASSERT_EQ(res["mdis"]["ck"]["deps"]["objs"].size(), 2);
+  ASSERT_EQ(res["mdis"]["spk"]["reconstructed"].size(), 2);
+  ASSERT_EQ(res["mdis"]["spk"]["deps"]["lsk"].size(), 2);
+  ASSERT_EQ(res["mdis"]["tspk"]["na"].size(), 1); 
+  ASSERT_EQ(res["mdis"]["fk"].size(), 2);
+  ASSERT_EQ(res["mdis"]["ik"].size(), 2);
+  ASSERT_EQ(res["mdis"]["iak"].size(), 2);
+  ASSERT_EQ(res["mdis"]["pck"].size(), 2);
```
Comment (Kelvinrr) [inline SugarSpice/tests/QueryTests.cpp:25]: I think expected counts are good enough for now. The individual components need to be unit tested separately in another PR. This was mostly about setting up ground work for testing the config files.

**Comment by Kelvinrr** [comment]
Ugh, for some reason the directories are not being generated on the remote tests.

**Comment by Kelvinrr** [comment]
Cursed PR finally passing

**Review by AustinSanders — APPROVED** [review]
Overall looks good.

I didn't leave inline comments because I think it's everywhere throughout our codebase, but 'using namespace std' is typically viewed as bad practice, so it might be worth trying to start moving away from it?  Doesn't matter much to me either way as long as it's consistent with what we've been doing.

**Comment by Kelvinrr** [comment]
Using namespace std is only really bad practice in headers, this is because it causes the namespace to be injected into anyone's code that imports your header. If you see it in any headers, point it out and I'll remove it. 

Testing headers matter really as those don't get shipped.
